tests/Unit/BaseProductPackagingTypeTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Unit;
+
+use Kollex\Assortment\BaseProductPackagingType;
+use PHPUnit\Framework\TestCase;
+
+final class BaseProductPackagingTypeTest extends TestCase
```

Comment:
if we are using final declaration it also makes sense to use it in our previous code where it can be useful